RomanASP.py

```diff
@@ -140,14 +156,18 @@ def main():
     airy = galsim.ChromaticOpticalPSF(lam, diam=2.36,
                                       aberrations=aberrations)
 
+    Lager.warning('manually removing weird simulation leftovers for now')
```

Comment:
I don't understand what this would mean if I were reading a log file and say this line.  This seems like more of a comment about the code rather than a log message for the running.  What is this warning me of?